src/history.rs

```diff
@@ -531,13 +739,14 @@ impl<'a> DynamicHistory<'a> {
         Ok(())
     }
 
+    // NOTE: if there was an associated paired wall clock time entry
+    // (via snapshot merge/produce_with_time), it'll precede the logical clocks
```

Comment:
Seems like this information belongs on the `merge_internal` function as well -- the fact that in the log the order will end up something like "(wct)?local,remote"